recipes/AddRemoveEventListenerJs/src/Main.purs

```diff
@@ -85,7 +85,7 @@ main = do
       -- In this approach, we use a better approach to add an event listener
       -- where we don't need to store a reference to the listener since
       -- it is still in scope.
-      removeListener <- addBetterListener METypes.click false buttonMain \e -> do
+      removeListener <- addBetterListener METypes.click false buttonMain \_e -> do
```

Comment:
Shouldn't this just be `\_ ->` if the `e` isn't used?

Reply:
Unused, named parameters as feature was intentionally used here, to give a bit of context what's ignored (e.g. `event`, `rec` and so on). I can drop it and use `_` instead.